resources/views/event/display_includes/tickets.blade.php

```diff
@@ -147,33 +160,87 @@ class="card-link text-info">
 
             </div>
 
+            {{-- 5 cases (pp = prepaid, npp = not prepaid) 
+                1: pp, npp and fees
+                2: pp, npp and no fees
+                3: pp, and fees
+                4: pp and no fees
+                5: npp    
+            --}}
             @if(Auth::check() && $tickets_available > 0)
             <div class="card-footer">
-
-                <button type="submit" class="btn btn-success btn-block"
-                       onclick="return confirm('You are about to buy €'+total.toFixed(2)+' worth of tickets. Are you sure?')">
-                    Total: <strong>&euro;<span id="ticket-total" class="mr-3">0.00</span></strong> Finish purchase!
-                </button>
-
+                {{-- No fees of no prepaid (2,4,5) --}}
+                @if (!config('omnomcom.mollie.use_fees') || !$has_prepay_tickets)
+                    <button type="submit" class="btn btn-success btn-block">
+                        Total: <strong>&euro;<span id="ticket-total" class="mr-3">0.00</span></strong> Finish purchase!
+                    </button>
+                {{-- fees and only prepaid (3) --}}
+                @elseif (config('omnomcom.mollie.use_fees') && $only_prepaid)
+                    @include('event.display_includes.mollie-modal')
+                    <a href="javascript:void();" class="btn btn-primary btn-block" data-toggle="modal" data-target="#mollie-modal">
+                        Get tickets now!
+                    </a>
+                @else
+                    <button id="directpay" type="submit" class="btn btn-success btn-block">
+                        Total: <strong>&euro;<span id="ticket-total" class="mr-3">0.00</span></strong> Finish purchase!
+                    </button>
+                    @include('event.display_includes.mollie-modal')
+                    <a hidden id="feesbutton" href="javascript:void();" class="btn btn-primary btn-block" data-toggle="modal" data-target="#mollie-modal">
+                        Get tickets now!
+                    </a>
+                @endif
             </div>
             @endif
-
         </div>
 
     </form>
 
     <script type="text/javascript" nonce="{{ csp_nonce() }}">
+        console.log("test")
+        var total = 0;
+        
 
-        let total = 0;
-
+        document.querySelectorAll(".ticket-select").forEach(ticket=>ticket.addEventListener('change', updateOrderTotal))
         function updateOrderTotal() {
             total = 0;
+            console.log("hi");
             $('.ticket-select').each(function () {
                 total += $(this).attr('data-price') * $(this).val();
             });
-            $('#ticket-total').html(total.toFixed(2))
+            $('#ticket-total').html(total.toFixed(2));
         }
 
     </script>
 
+    @if(!(config('omnomcom.mollie.use_fees') && $only_prepaid) && !(!config('omnomcom.mollie.use_fees') || !$has_prepay_tickets))
```

Comment:
I don't understand why this all needs to be negated, can you either check the not use fees first or make the second half not a negation of a negated or?